spec/jobs/reindex_model_job_spec.rb

```diff
@@ -14,11 +29,10 @@
     Apartment::Tenant.switch!(account.tenant) { work }
   end
 
-  it "#perform_later can enqueue model for reindex later" do
-    expect { described_class.perform_later(work.class.to_s, account.cname) }.to have_enqueued_job(described_class).exactly(:once)
-  end
-
-  it "#perform_now will enqueue model for reindex" do
-    expect { described_class.perform_now(work.class.to_s, account.cname) }.to have_enqueued_job(described_class).exactly(:once)
+  it "sets doi_status_when_public to findable" do
+    expect(work.doi).to be_empty
```

Comment:
We don't need to test this, it is defined as empty on line 5.

Reply:
@thenapking  despite I discovered sometimes the work in the test I removed that was using **second_work** still had doi set, so this was added just for me to be double sure the doi is empty or it fails.

Reply:
@thenapking  despite that, I discovered that sometimes for the work in the test I removed that was using **second_work** the doi was set, when ought not to be so this was added just for me to be double sure tat the point of triggering the job that he doi was indeed empty otherwise the test  should fail.

I am happy to remove though.